packages/nodes-base/nodes/Webflow/WebflowTrigger.node.ts

```diff
@@ -9,7 +9,7 @@ import type {
 	IWebhookResponseData,
 } from 'n8n-workflow';
 
-import { webflowApiRequest } from './GenericFunctions';
+import { webflowApiRequest } from './V1/GenericFunctions';
```

Comment:
related to the first comment, I think we should avoid referencing files within the version folder

Reply:
This is referenced while I look at the Trigger node in another PR as it could be trickier to move depending on how Node versioning is at the moment with trigger nodes, Last time I tried it the versioning wasn't working so this would at least keep the trigger node working until we get a chance to fix it.